### DISCUSSION TITLE
Convert Text to Polylines: How do I set the location?

### DISCUSSION THREAD

**Discussion by Artyom-Murzinov** [mozman/ezdxf#1192] [Q&A]
Вопрос как установить координаты текста в этом коде по умолчанию (X=0, Y=0) а хотелось бы смещать текст 

The question is how to set the text coordinates in this code by default (X=0, Y=0) but I would like to shift the text

```Python
import ezdxf
from ezdxf.fonts.fonts import FontFace
from ezdxf.path import render_lwpolylines
from ezdxf.addons.text2path import make_path_from_str
from ezdxf.enums import TextEntityAlignment
support_dirs =  "/Users/artemmurzinov/Downloads/Шрифты",
ezdxf.options.support_dirs
doc = ezdxf.new("R2018")
msp = doc.modelspace()
text = "почему-бы шрифт не сместить?"

p = make_path_from_str(text, FontFace(filename ="GOST_AU.shx", style="Oblique", weight=0), 
                                       size=5,
                                       length = 0, 
                                       align=TextEntityAlignment.TOP_CENTER
                                       )

render_lwpolylines(msp,[p])

with open("out.dxf", "w") as f:
    doc.write(f)
```

**Comment by mozman** [comment]
The function `make_path_from_str()` returns `Path` objects, which can be transformed by `ezdxf.path.transform_paths()`.

**Comment by Artyom-Murzinov** [comment]
я сделал так:
text_content = "что-то уже получается"
text_size = 15.0 # Example size
text_position = (5, 10) # Example position
doc.styles.new('MyTextStyle', dxfattribs={'font': 'GOST_A.TTF'})
# Add text to the DXF document
dxf_poli = msp.add_text(text_content, dxfattribs = {
   'height': text_size,
   'style' :'MyTextStyle'
}).set_placement(text_position, align=TextEntityAlignment.CENTER)
engraving = explode(dxf_poli, kind=4)
